app/screens/backup/Backup.tsx

```diff
@@ -12,7 +13,11 @@ const Wrapper = styled.div`
 
 const Backup = ({ history }: RouteComponentProps) => (
   <Wrapper>
-    <BackButton action={history.goBack} width={7} height={10} />
+    <BackButton
+      action={() => history.push(MainPath.Wallet)}
```

Comment:
Why do we `push` here (and in the code below) instead of `replace`?

It seems to me that you've started using `replace` as a common practice for "Back" buttons. And it sounds reasonable: since we're going "Back" we don't need to store previous history path. And it seems that it is the best practice to have a properly working `history.goBack()`

So I want to understand why in some cases you're doing `push` and `replace` in others. So if there is nothing special, I'd prefer to replace all `history.push` with `history.replace` for any "Back" button, to have a consistent history behavior.
Let's say that the User has such a history: `A -> B -> C`, and then he goes back to `B` (via `history.push`), in case screen B has a button that calls `history.go Back()` it will redirect us to `C` instead of `A`.